ui/webui/src/components/storage/DiskEncryption.jsx

```diff
@@ -206,19 +208,17 @@ const getRuleLength = (password) => {
 const getRuleConfirmMatches = (password, confirm) => (password.length > 0 ? (password === confirm ? "success" : "error") : "indeterminate");
 
 const CheckDisksSpinner = (
-    <Bullseye>
```

Comment:
You need to remove also the redundant import now :
22:5  error  'Bullseye' is defined but never used  no-unused-vars

Reply:
Thanks!